src/components/logout/logout_container.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```

Comment:
Done. Now, each component with a controller should look like
```
components/
    button/
        button.js
        button_container.js
        index.js
```
so we can always import it as `components/button` regardless of whether or not it has a container.